### PR TITLE
fix(tuto): matomo + timescale MTA-1688e

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: tutorials/matomo-analytics/index.mdx [hunk 1]
```diff
@@ -30,55 +30,55 @@ The tool is written in PHP and stores its data in a MySQL/MariaDB database.
 
 ## Installing the LEMP-Stack
 
-Matomo is a web application written in [PHP](https://php.net/) and uses [MySQL/MariaDB](https://mariadb.org/) database to store its data. To serve the web contents, it relies on a web server, for example [Nginx](http://nginx.org/). Therefore a working [LEMP-Stack](/tutorials/installation-lemp-ubuntu-bionic) is required on the Matomo instance.
+Matomo relies on a web server, for example [Nginx](http://nginx.org/). A working [LEMP-Stack](/tutorials/installation-lemp-ubuntu-bionic) must be set up on your Instance for Matomo to run.
 
-1. Update the apt packet cache and upgrade the packets already installed on the instance:
+1. Update the apt packet cache and upgrade the packets already installed on the instance.
```
Comment (RoRoJ) [inline tutorials/matomo-analytics/index.mdx:35]:
```suggestion
1. Update the apt packet cache and upgrade the packets already installed on the Instance.
```

File: tutorials/matomo-analytics/index.mdx [hunk 2]
```diff
@@ -88,17 +88,17 @@ Matomo is a web application written in [PHP](https://php.net/) and uses [MySQL/M
   
 <Message type="note">
 
-  When doing the following steps, replace `matomo.example.com` with the domain name your Matomo will run on.
+  When following the next steps, replace `matomo.example.com` with your domain name.
 
 </Message>
 
-1. Create a Nginx Configuration file and open it in a text editor:
+1. Create a Nginx Configuration file and open it in a text editor.
```
Comment (RoRoJ) [inline tutorials/matomo-analytics/index.mdx:95]:
```suggestion
1. Create a Nginx configuration file and open it in a text editor.
```

File: tutorials/matomo-analytics/index.mdx [hunk 3]
```diff
@@ -203,9 +206,9 @@ Matomo is a web application written in [PHP](https://php.net/) and uses [MySQL/M
     - Tick the first box to receive notifications about Matomo updates by email (optional)
     - Tick the second box if you want to receive commercial information from Matomo by email (optional)
 
-  Click **Next** to create the super user account and to proceed with the installation.
+  Click **Next** to create the super user account and proceed with the installation.
 
-6. Setup a first website that shall be tracked and analysed by Matomo:
+6. Set up the first website you want Matomo to track and analyse.
```
Comment (RoRoJ) [inline tutorials/matomo-analytics/index.mdx:211]:
```suggestion
6. Set up the first website you want Matomo to track and analyze.
```

File: tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx [hunk 4]
```diff
@@ -5,63 +5,36 @@ meta:
...
+[TimescaleDB](https://www.timescale.com/) is a time series database built on [PostgreSQL](/tutorials/install-postgresql/).  
+A time-series database typically stores “time-series data”, as the name indicates. It can be seen as a sequence of data points, that measure the same thing over time and store the measurement results in time order.  
 Each of these measurements is paired with a timestamp, defined by a name and a set of labeled dimensions (or “tags”).
 
-Time-series data is everywhere around us and an important point of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains.
+Time-series data is everywhere around us and an important part of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains.
```
Comment (RoRoJ) [inline tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx:20]:
```suggestion
Time-series data is everywhere around us and an important part of our everyday life. Whether it is a factory that measures the production output of a specific machine, a farmer observing the humidity of the soil or a city measuring the regularity between trains, all of this is time-series data.
```

Comment (RoRoJ) [reply tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx:20]: I think gramatically either it needs something at the end, or it needs to start with something other than "whether"

File: tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx [hunk 5]
```diff
@@ -187,9 +160,9 @@ The following steps assume that you have already created a Database Instance. Fo
 
   Manually adding data works for a small scale database, but when it comes to deal with time-scale data you have large sets of data.
 
-### Importing a lagrge dataset
+### Importing the dataset
 
-To have a large dataset available we are going to use the [air quality data](https://dataratp2.opendatasoft.com/explore/dataset/qualite-de-lair-mesuree-dans-la-station-franklin-d-roosevelt/information/) from a parisian metro station. The data is provided by Paris' transport authority [RATP](https://dataratp2.opendatasoft.com/explore/?sort=modified)
+In this tutorial, we use an [air quality dataset](https://dataratp2.opendatasoft.com/explore/dataset/qualite-de-lair-mesuree-dans-la-station-franklin-d-roosevelt/information/) from a parisian metro station. The data is provided by Paris' transport authority [RATP](https://dataratp2.opendatasoft.com/explore/?sort=modified).
```
Comment (RoRoJ) [inline tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx:165]:
```suggestion
In this tutorial, we use an [air quality dataset](https://dataratp2.opendatasoft.com/explore/dataset/qualite-de-lair-mesuree-dans-la-station-franklin-d-roosevelt/information/) from a Parisian metro station. The data is provided by Paris' transport authority [RATP](https://dataratp2.opendatasoft.com/explore/?sort=modified).
```

File: tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx [hunk 6]
```diff
@@ -235,7 +208,7 @@ Now as you have a large dataset available, you can query the data using the `psq
   (10 rows)
   ```
 
-3. To retrieve the average temperature on all data run the following command:
+3. Run the following command, to retrieve the average temperature on all data:
```
Comment (RoRoJ) [inline tutorials/visualize-timeseries-data-timescaledb-grafana/index.mdx:211]:
```suggestion
3. Run the following command to retrieve the average temperature on all data:
```

**Review by RoRoJ — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

**Pull request merged**